### PR TITLE
Add Postgres/MySql/ManagedInstance online migration scenarios

### PR DESCRIPTION
**Pull request by ash1625**
## Description
REST spec PR link to the SDK PR :
Azure/azure-rest-api-specs#4627

This PR adds the below new scenarios to Azure Database Migration Service:
1) SQL to Azure SQL Managed Instance online migration
2) SQL to Azure Postgres SQL online migration 
3)  SQL to Azure MySQL online migration

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by ash1625** [comment]
REST spec PR link to the SDK PR : 
https://github.com/Azure/azure-rest-api-specs/pull/4627

**Review by hitenjava — APPROVED** [review]

File: src/SDKs/DataMigration/Management.DataMigration/Microsoft.Azure.Management.DataMigration.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DataMigration</PackageId>
 		<Description>Provides developers with libraries for the Database Migration Service (DMS) under Azure Resource manager to deploy an instance of DMS. Create, Delete, Read Projects, Files and Tasks and more.</Description>
-		<Version>0.6.0-preview</Version>
+		<Version>0.7.0-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.DataMigration</AssemblyName>
 		<PackageTags>DataMigration;Data Migration;DataMigration_2018-07-15-preview;</PackageTags>
 		<PackageReleaseNotes>Preview release of the Database Migration Service (DMS) SDK to Create, Delete, Read Projects, Files and Database Migration Tasks.</PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/DataMigration/Management.DataMigration/Microsoft.Azure.Management.DataMigration.csproj]: Please update `PackageReleaseNotes`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part, needs a minor change,

File: src/SDKs/DataMigration/Management.DataMigration/Microsoft.Azure.Management.DataMigration.csproj [hunk 2]
```diff
@@ -9,7 +9,7 @@
 		<Version>0.7.0-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.DataMigration</AssemblyName>
 		<PackageTags>DataMigration;Data Migration;DataMigration_2018-07-15-preview;</PackageTags>
-		<PackageReleaseNotes>Preview release of the Database Migration Service (DMS) SDK to Create, Delete, Read Projects, Files and Database Migration Tasks.</PackageReleaseNotes>
+		<PackageReleaseNotes>Preview release of the Database Migration Service (DMS) SDK to Create, Delete, Read Projects, Files and Database Migration Tasks. The SDK now supports multiple online scenarios for targets like Azure SQL Managed Instance, Azure Postgres and Azure MySql</PackageReleaseNotes>
```
Comment (hitenjava) [inline src/SDKs/DataMigration/Management.DataMigration/Microsoft.Azure.Management.DataMigration.csproj]: "PostgreSQL"
"MySQL"

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**